contracts/src/Teleporter/upgrades/TeleporterRegistry.sol

```diff
@@ -177,14 +177,12 @@ contract TeleporterRegistry {
      *
      * Emits a {AddProtocolVersion} event when successful.
      * Emits a {LatestVersionUpdated} event when a new protocol version greater than the current latest version is added.
-     * Note: `protocolAddress` doesn't have to be a contract address, this is primarily
-     * to support the case of registering a new protocol address meant for a security patch
-     * before the contract is deployed, and prevent the vulnerabilitiy from being exposed before the registry update.
+     * Note: `entry.protocolAddress` doesn't have to be a contract address, allowing a new protocol address to be registered before the contract is deployed. This may be the desired order of events to prevent publishing the patch for a vulernability in the current version before the registry is updated to the new version.
```

Comment:
I don't love the use of the word "vulnerability" here, as it could cause unnecessary FUD. I think we should remove the second sentence altogether, and not speculate too much on use cases.
```suggestion
     * Note: `entry.protocolAddress` doesn't have to be a contract address, allowing a new protocol address to be registered before the contract is deployed.
```

Reply:
hmm ye I'm good with the first line covering how the registry can be used.